data-model/fdo-type/shared-model/0.3.0/schema/citation.json

```diff
@@ -33,7 +33,8 @@
     },
     "dcterms:creator": {
```

Comment:
if we're not using the dcterms definition anymore, is it still in the dcterms namepsace?